### DISCUSSION TITLE
Should the Weld feature be a separate app?

### DISCUSSION THREAD

**Discussion by SilkieSabra** [OpenCollarTeam/OpenCollar#1009] [Ideas]
We are preparing to move the Weld feature to a separate, modular app that can be installed and removed separately.  This pertains to pr #944 , read the details there.  The advantages to this would be 1) it would take a couple more steps to weld a collar in the first place, reducing accidents; 2) it would integrate the unwelding process so no external unwelder is required; and 3) it would open up space in oc_core and oc_settings, both of which need the space.  What do you think?

**Comment by Medea-Destiny** [comment]
As it currently stands, everything operates from within the plugin, which is an optional component in the collar. That means you can be welded, then remove the weld plugin, and you'll stay welded. You'd be able to add it back in again to unweld.

One subject of discussion is the immersion value of having some kind of physical welder/unwelder object which you  have to be in the presence of for it to work. Thoughts?

**Comment by MinesweeperM** [comment]
I guess, would be good to make both welding and unwelding process, a little time consuming. 
For now, unwelding is confusing process, and from the other side, if you know what to do, it is fast and easy. Too fast and too easy.
So some physical device to weld, where sub and owner will need to sit for 10-15 minutes, and to unweld, sub and owner to sit there for another one 10-15 minutes, or sub and NOT owner - 2-3 hours, would be nice

**Comment by Pingout** [comment]
Maybe what @MinesweeperM is suggesting could be done, but it would need to include stuff way outside of our little tent. 
One could make a device which would aparently hold the Submissive (similar to a dentist chair), and then the Owner or Helper would also need a pose, holding tools etc., and then the Unwelder script could take over and perform the unweld, while Spark Particles fly all around.  For "10-15" minutes! Set the timer for between 600 and 900? Oh and OSHA Approved Safety glasses required.
Marketplace has dozens of Collaring Ritual devices. This would be an Uncollaring Ritual device.

**Comment by Medea-Destiny** [comment]
A thought -- should we have a button in the app menu to remove the app script? That might be useful as the app is no longer needed unless someone wants to unweld, and it could help give people that feeling of finality (though of course they can just reinstall the oc_weld app from an installer

**Comment by Medea-Destiny** [comment]
On @MinesweeperM 's suggestion -- It's a nice idea, but my concern here is that if we've got people heading over to the HQ to use an unwelder, we could get people waiting in a queue. Maybe as a compromise, we could have the unwelder require the collar wearer to stand close to it for a few minutes while sparks fly from their collar, but not actually sit on it and occupy it, so multiple people could use it at the same time.

**Comment by MinesweeperM** [comment]
Well, maybe it could rez welding/unwelding tool by addon, so people will not be tied to HQ...
My main idea, is to make process simple, to avoid concourse of confused people, who weild, and can't understand, how to unweld; and in same time, time and efforts expensive, to make feel it really welded...

**Comment by Pingout** [comment]
As a way to resolve a potential backlog of Collar Wearers standing in line, waiting to use the Unwelder.... We could just rez another Unweld Station or two. In fact I have an Unwelder at my collar store, available for anyone who needs to use it.
And, we already have the Unwelder HUD, which anyone can just grab at the OC HQ
Regardless, I will play around with adding animations to the unwelder device, to see if we can create an Uncollaring Ritual Device.
